src/eval/tests/trie.rs

```diff
@@ -10,34 +10,29 @@ fn trie_lang() {
 
     install(s, &mut lang);
 
-    let expr = "(let ((trie (.lurk.trie.new)))
+    // TODO wrap .lurk.trie.new, .lurk.trie.lookup and .lurk.trie.insert in lambdas
```

Comment:
I think this should be handled at the coprocessor level. Having this test case is good because it's an example of what the ux should be like for a coprocessor user (here) and implementer (elsewhere in the Trie implementation, which should *not* require significant — or ideally *any* — changes to absorb the corrected/improved coprocessor behavior.